internal/auth0/organization/resource_member.go

```diff
@@ -57,15 +57,15 @@ func readOrganizationMember(ctx context.Context, data *schema.ResourceData, meta
 	api := meta.(*config.Config).GetAPI()
 
 	organizationID := data.Get("organization_id").(string)
+	userID := data.Get("user_id").(string)
```

Comment:
We appear to have broadened the scope of the `userID` by defining it higher up than necessary. To minimize the risk of errors from accessing a variable in an expanded scope, it's best practice to keep variables as close to their point of use as possible.